community-membership.md

```diff
@@ -157,5 +158,69 @@ The following apply to the subproject(repos) for which one would be an owner:
 - Ensure a healthy process for discussion and decision making is in place
 - Work with other maintainers to maintain the project's overall health and success holistically
 
-Maintainers *MUST* remain active. If they are unresponsive for >3 months, they will be automatically removed unless a super-majority of the other repository maintainers agrees to extend the period to be greater than 3 months.
+Core-Maintainers *MUST* remain active. If they are unresponsive for >3 months, they will be automatically removed unless a super-majority of the other repository maintainers agrees to extend the period to be greater than 3 months.
 
+## Feature-Maintainer
+
+Feature-Maintainers are the technical authority for a subarea of the project/repo in the Dapr org. They *MUST* have demonstrated both good judgement and responsibility towards the health of that subarea. Feature-maintainers can set technical direction and make or approve design decisions for their subarea - either directly or through delegation of these responsibilities.
+
+Defined by: GitHub organization ownership, permissions and entry in `CODEOWNERS`
+files.
+
+### Requirements
+
+The following apply to the sub area(ex: building block API/ Control plane service/ Component provider) for which one would be a Feature-maintainer:
+
+- Deep understanding of the technical goals and direction of the sub area.
+- Deep understanding of the technical domain (specifically the language) of the sub area
+- Sustained contributions to design and direction by doing all of:
+  - Authoring and reviewing proposals
+  - Initiating, contributing and resolving discussions  For example Discord discussions,  GitHub issues, meetings)
+  - Identifying subtle or complex issues in designs and implementation PRs
+- Directly contributed to the subarea through implementation and / or review
+- Must have been active for 3 months or more for the given sub-area
+- Inactivity for more than 3 months leads to a removal vote by the repo maintainers and not an automatic removal
+
+
+### Acceptance
+
+  New Feature-Maintainers can be added to the project by a super-majority (two-thirds / 66.66%) vote. Only the maintainers of the repository in which the nomination is applied have a binding vote, while maintainers from other repositories are on an informed basis via a separate email thread.
```

Comment:
Does maintainers voting here imply only core maintainers or core and feature maintainers? I presume this is still core maintainers only.